helpers/env-variables.ts

```diff
@@ -13,6 +13,12 @@ import {
 
 import { TSYSTEMS_LLMHUB_API_URL } from "./providers/llmhub";
 
+const DEFAULT_SYSTEM_PROMPT =
+  "You are a helpful assistant who helps users with their questions.";
```

Comment:
How about we instead remove the specific system prompts for Wikipedia and duckduckgo tool?

Reply:
Great idea, I think we can remove their system prompt because we don't need any specific tuning to use the tools

Reply:
Might still be helpful to prioritize the query engine tool somehow if it's selected. E.g. with the right tool description